apps/test/unit/gamelab/animationListModuleTest.js

```diff
@@ -27,39 +27,68 @@ describe('animationListModule', function () {
   describe('animationSourceUrl', function () {
     const key = 'foo';
 
-    it(`returns the sourceUrl from props if it exists`, function () {
-      const props = {sourceUrl: 'bar'};
-      expect(animationSourceUrl(key, props)).to.equal('bar');
-      expect(animationSourceUrl(null, props)).to.equal('bar');
+    it(`returns the sourceUrl from props if it exists and is not an uploaded image`, function () {
+      const animationList = {orderedKeys: ['foo'], propsByKey: {'foo' : {sourceUrl: 'bar'}}};
+      expect(animationSourceUrl(key, animationList, '123')).to.equal('bar');
+    });
+
+    it(`returns the sourceUrl from props if it exists and contains the version`, function () {
+      const animationList = {orderedKeys: ['foo'], propsByKey: {'foo' : {sourceUrl: '/v3/animations/test?version=foo'}}};
+      expect(animationSourceUrl(key, animationList, '123')).to.equal('/v3/animations/test?version=foo');
```

Comment:
Nit: Using `version=foo` in this test when the animation key is also foo makes it somewhat possible to conflate the animation key and version id when reading this test... and wouldn't catch the (unlikely) bug that put the animation key where the version should be.  Better to use a different string for each.